pkg/kubectl/cmd/cp.go

```diff
@@ -195,6 +195,11 @@ func makeTar(filepath string, writer io.Writer) error {
 	// TODO: use compression here?
 	tarWriter := tar.NewWriter(writer)
 	defer tarWriter.Close()
+
+	len := len(filepath)
+	if filepath[len-1] == '/' {
+		filepath = filepath[0 : len-1]
```

Comment:
`0` can be omitted.

Reply:
ok

Reply:
Will this panic if len=0?

Reply:
What about: strings.HasSuffix(filepath, "/")